back/engines/commercial/id_criipto/spec/requests/criipto_verification_spec.rb

```diff
@@ -102,7 +102,8 @@
         municipality_code_custom_field_key: 'municipality_code',
         method_name_multiloc: {
           en: 'MitID Verification'
-        }
+        },
+        uid_field_pattern: '%{sub}'
```

Comment:
All the old tests pass with this config. It ensures the feature will continue working as before.